### ISSUE TITLE
Unexpected char '<' at 1:1 (JSON::ParseException)

### ISSUE THREAD

**Issue by boogsnaxx** [iv-org/invidious#1535]
Error occurs when importing OPML subscriptions data.

Date: `2020-12-05T09:23:59Z`
Route: `/data_control?referer=%2Ffeed%2Fsubscriptions`
Version: `-ca4408b @ master`
<details>
<summary>Backtrace</summary>
<p>
   
```
Unexpected char '<' at 1:1 (JSON::ParseException)
  from ../usr/share/crystal/src/json/lexer.cr:371:7 in 'raise'
  from ../usr/share/crystal/src/json/lexer.cr:367:5 in 'unexpected_char'
  from ../usr/share/crystal/src/json/lexer.cr:366:11 in 'unexpected_char'
  from ../usr/share/crystal/src/json/lexer.cr:247:7 in 'consume_number'
  from ../usr/share/crystal/src/json/lexer.cr:63:7 in 'next_token'
  from ../usr/share/crystal/src/json/parser.cr:108:11 in 'next_token'
  from ../usr/share/crystal/src/json/parser.cr:9:5 in 'initialize'
  from ../usr/share/crystal/src/json/parser.cr:6:3 in 'new'
  from ../usr/share/crystal/src/json.cr:131:5 in 'parse'
  from src/invidious.cr:1427:16 in '->'
  from ../usr/share/crystal/src/primitives.cr:255:3 in '->'
  from ../usr/share/crystal/src/primitives.cr:255:3 in 'process_request'
  from lib/kemal/src/kemal/route_handler.cr:17:7 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/websocket_handler.cr:13:14 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:212:5 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:94:12 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:145:12 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:68:12 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/static_file_handler.cr:112:11 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/exception_handler.cr:8:7 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/logger.cr:16:5 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'call'
  from ../usr/share/crystal/src/http/server/request_processor.cr:50:11 in 'process'
  from ../usr/share/crystal/src/http/server.cr:513:5 in 'handle_client'
  from ../usr/share/crystal/src/http/server.cr:468:13 in '->'
  from ../usr/share/crystal/src/primitives.cr:255:3 in 'run'
  from ../usr/share/crystal/src/fiber.cr:92:34 in '->'
  from ???
```
</p>
</details>

**Comment by Wyrrrd** [comment]
Might be connected to #783

**Comment by fareszr** [comment]
i have the same issue currently

logs:
Title: `Unexpected char '<' at 1:1 (JSON::ParseException)`
Date: `2020-12-25T22:42:32Z`
Route: `/channel/UCs6KfncB4OV6Vug4o_bzijg`
Version: `2020.12.23-eeeecf9 @ master`
<details>
<summary>Backtrace</summary>
<p>
   
```
Unexpected char '<' at 1:1 (JSON::ParseException)
  from ../usr/share/crystal/src/json/lexer.cr:371:7 in 'raise'
  from ../usr/share/crystal/src/json/lexer.cr:367:5 in 'unexpected_char'
  from ../usr/share/crystal/src/json/lexer.cr:366:11 in 'unexpected_char'
  from ../usr/share/crystal/src/json/lexer.cr:247:7 in 'consume_number'
  from ../usr/share/crystal/src/json/lexer.cr:63:7 in 'next_token'
  from ../usr/share/crystal/src/json/parser.cr:108:11 in 'next_token'
  from ../usr/share/crystal/src/json/parser.cr:9:5 in 'initialize'
  from ../usr/share/crystal/src/json/parser.cr:6:3 in 'new'
  from ../usr/share/crystal/src/json.cr:131:5 in 'parse'
  from src/invidious/channels.cr:926:20 in 'get_60_videos'
  from src/invidious.cr:1698:20 in '->'
  from ../usr/share/crystal/src/primitives.cr:255:3 in '->'
  from ../usr/share/crystal/src/primitives.cr:255:3 in 'process_request'
  from lib/kemal/src/kemal/route_handler.cr:17:7 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/websocket_handler.cr:13:14 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:212:5 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:94:12 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:145:12 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/handlers.cr:70:5 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/static_file_handler.cr:189:11 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/exception_handler.cr:8:7 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from src/invidious/helpers/logger.cr:19:35 in 'call'
  from ../usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'call'
  from ../usr/share/crystal/src/http/server/request_processor.cr:50:11 in 'process'
  from ../usr/share/crystal/src/http/server.cr:513:5 in 'handle_client'
  from ../usr/share/crystal/src/http/server.cr:468:13 in '->'
  from ../usr/share/crystal/src/primitives.cr:255:3 in 'run'
  from ../usr/share/crystal/src/fiber.cr:92:34 in '->'
  from ???
```
</p>
</details>

i cant view any channels however playing videos is fine

**Comment by TheFrenchGhosty** [comment]
Fixed in #1633
